atlp-rwanda/e-commerce-lydia-32-fn
# New Notification Toast

### What does this PR do?

- This PR implements real-time notifications for users, ensuring that toast notifications are displayed whenever new notifications are received. It enhances the user experience by providing immediate feedback on new notifications without requiring a page refresh or manual data refetch.

### How should this be manually tested?

1. Run the Application: Start the application and ensure that it is connected to the WebSocket or real-time service.
2. Trigger New Notifications: Use the application or an external tool to simulate the arrival of new notifications.
3.Verify Toast Notifications:
    - Check that a toast notification appears for each new notification received.
    - Ensure that the toast notifications are styled according to the custom design.
    - Confirm that no duplicate toasts are shown for the same notification.
    - Refresh and Reconnect: Reload the page or disconnect/reconnect to the real-time service to ensure that notifications are 
    correctly handled across sessions.

### What are the relevant pivotal tracker stories
https://www.pivotaltracker.com/story/show/187570833
